### PR TITLE
adds hint and updates helptext for PAC

### PR DESCRIPTION
**Pull request by invincibleJai**
**Fixes**: 
https://issues.redhat.com/browse/ODC-6673

**Solution Description**: 

Add a help text to the GitHub application name form to give more context to the user and show a hint on the overview page for the returning users.

**Screen shots / Gifs for design review**: 

- Hint for returning user

image

- Help text for GitHub App name

image

**Test setup:**

- Install Pipelines operator
- Go to Admin -> pipelines list page and click on Setup GitHub App
- Enter the app name and press `Setup`

**Unit Tests:**

image

**Browser conformance**: 

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: frontend/packages/pipelines-plugin/src/components/pac/PacOverview.tsx [hunk 1]
```diff
@@ -95,6 +97,20 @@ const PacOverview: React.FC<PacOverviewProps> = ({
...
+          {!showSuccessAlert && annotations?.appUrl && (
+            <Hint>
+              <HintBody>
+                <Trans t={t} ns="pipelines-plugin">
+                  Use the{' '}
+                  <a href={annotations.appUrl} target="_blank" rel="noopener noreferrer">
+                    link
+                  </a>{' '}
+                  to install the GitHub Application to your repositories in your
+                  organization/account
```
Comment (karthikjeeyar) [inline frontend/packages/pipelines-plugin/src/components/pac/PacOverview.tsx:109]: nit
```suggestion
                  organization/account.
```

File: frontend/packages/pipelines-plugin/src/components/pac/PacAppForm.tsx [hunk 2]
```diff
@@ -41,7 +41,9 @@ const PacAppForm: React.FC<FormikProps<FormikValues>> = ({
           <FormGroup
             label={t('pipelines-plugin~GitHub application name')}
             isRequired
-            helperText={t('pipelines-plugin~Provide a unique name')}
+            helperText={t(
+              'pipelines-plugin~Provide a unique name to your GitHub app e.g. "pipelines-ci-clustername" ',
```
Comment (jerolimov) [inline frontend/packages/pipelines-plugin/src/components/pac/PacAppForm.tsx:45]: Is this space needed? :smirk: 

```suggestion
              'pipelines-plugin~Provide a unique name to your GitHub app e.g. "pipelines-ci-clustername" ,
```

**Comment by MariaLeonova** [comment]
This looks great! Small language correction : 

- Hint for returning user:
it should say "Use this link" and have a full-stop at the end of the sentence.

- Help text for GitHub App name:
   it should say "for your GitHub app", and also please put a comma before "e.g."

For app setup, the spacing seems a bit off between the title and the name, please reference this screen for margin:
Screenshot 2022-05-18 at 15 04 19
Screenshot 2022-05-18 at 15 06 51

**Comment by invincibleJai** [comment]
> This looks great! Small language correction :
> 
> * Hint for returning user:
>   it should say "Use this link" and have a full-stop at the end of the sentence.
> * Help text for GitHub App name:
>   it should say "for your GitHub app", and also please put a comma before "e.g."
> 
> For app setup, the spacing seems a bit off between the title and the name, please reference this screen for margin: Screenshot 2022-05-18 at 15 04 19 Screenshot 2022-05-18 at 15 06 51

Have fixed both the comments and updated screenshot, PTAL. Thanks!!

image

**Comment by karthikjeeyar** [comment]
Verified on clusterbot

image

/label qe-approved

Propagating other labels from the Epic https://issues.redhat.com/browse/ODC-6472

/label docs-approved
/label px-approved

**Pull request merged**